site/installing.md

```diff
@@ -53,11 +57,11 @@ Then, install Scope onto your local Docker machine and start using Weave Cloud:
 
 Where,
 
-* `--service-token=<token>` is the token that you obtained when you signed up.
+* `--service-token=<token>` is the token you obtained after you signed up for Weave Cloud.
```

Comment:
There are 3 occurances of this phrase (for k8s, and for compose).  If it consistent?